tests/countries/test_solomon_islands.py

```
@@ -0,0 +1,279 @@
+
+    def test_central_province_day(self):
+        observed_dts = (
+            "2013-06-28",
+            "2014-06-30",
+            "2019-06-28",
+            "2024-06-28",
+            "2025-06-30",
+        )
+        self._test_province_day_helper("CE", "Central Province Day", "06-29", observed_dts)
+
+    def test_choiseul_province_day(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)